kbs/src/attestation/mod.rs

```diff
@@ -40,6 +40,18 @@ pub async fn make_nonce() -> Result<String> {
     Ok(STANDARD.encode(&nonce))
 }
 
+pub(crate) async fn generic_generate_challenge(
+    _tee: Tee,
+    _tee_parameters: serde_json::Value,
+) -> Result<Challenge> {
```

Comment:
why does this have params if we don't use them? Should we maybe do `impl Default for Challenge`?

Reply:
I was trying to make this a minimal change and if you look at the existing code, you'll see the same unused params.

>  Should we maybe do impl Default for Challenge?

I thought of that, but `Challenge` is defined in the `kbs-types` crate, so that can be done at a later time imho.